test/tools/libtesteth/Options.cpp

```diff
@@ -229,7 +230,15 @@ Options::Options(int argc, const char** argv)
                 }
             }
             else
-                singleTestName = std::move(name1);
+            {
+                if (boost::filesystem::exists(name1))
```

Comment:
This could confusingly try to run a file in case there's a subdirectory or a file in the current dir with the same name as the test - the downside of using one CLI flag for several purposes